src/Raven.Client/Raven.Client.csproj

```diff
@@ -134,7 +134,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Lambda2Js.Signed" Version="3.1.4" />
-    <PackageReference Include="Microsoft.AspNetCore.JsonPatch" Version="7.0.18" />
+    <PackageReference Include="Microsoft.AspNetCore.JsonPatch" Version="8.0.4" />
```

Comment:
in this file, at the top there is a section that adds Sparrow.dll, you need to add net8 there as well